### ISSUE TITLE
[FR] Add a "known compatible with" field

### ISSUE THREAD

**Issue by khalwat** [pixelandtonic/craftnet#200]
Taking a page out of the WordPress ecosystem, it'd be useful for people using plugins to cast an affirmational vote that a plugin does work on various major versions of Craft CMS.

The display could be something like:

Known compatible with Craft CMS: 3.0, 3.1, 3.2, 3.3, 3.4

This would be a "crowd sourced" voting mechanism. The only issue I foresee could be lack of incentive for people to bother voting.

So you might need to have them listed as working by default, but allow people to vote **NO** if it's not working.

Whenever the author releases a new version of the plugin, this information is reset.

**Comment by evanwarner** [comment]
I'm _hugely_ in favor of doing this. Approving what plugins make it into the store is great, but once something is listed that's kinda it. Implementing this feature would offload some of the quality maintenance burden to the community so we can keep a collective eye on which plugins are keeping up with Craft's progress.

**Comment by bossanova808** [comment]
I agree, as the Craft world grows, this becomes really quite essential.

In addition, I will just express (again) that having an indication on the Updates page as to which updates involve a database migration and which don't would be **hugely** helpful.

For many sites, this probably isn't a big deal - but for live shops, migrations and when/how to update are a huge issue.  I think in general you would want folks to keep up to date.  Code only updates are _much_ easier for live systems to deal with.  DB ones need a lot more testing and, at least sometimes, a full site pause...which can be difficult.

**Comment by evanwarner** [comment]
@bossanova808 Your db migration thoughts probably belong in their own feature request, so they don't get lost/ignored here.

**Comment by bossanova808** [comment]
Indeed, you're right -> https://github.com/pixelandtonic/craftnet/issues/201

**Comment by niektenhoopen** [comment]
I would like to add that I would also like to see the (range of) PHP-compatibility of the plugin.

If it's compatible till PHP 7.4, I can't run it on PHP 8.
If it's compatible with PHP 5, it's probably not actively maintained and I would absolutely consider a different plugin because I cannot imagine someone not wanting to use typehints.
